src/shims/unix/fd.rs

```diff
@@ -233,12 +234,12 @@ impl FdTable {
         new_fd
     }
 
-    pub fn get(&self, fd: i32) -> Option<Ref<'_, dyn FileDescription>> {
```

Comment:
You can probably keep the `Option<Ref<'_, dyn FileDescription>>` return type by using `Ref::map`.

Reply:
learned a lot :)